FrontEnd/src/components/authorization/LoginContent.jsx

```diff
@@ -46,10 +50,36 @@ const LoginContent = (props) => {
       errorMessage = errors.email.message;
     } else if (errors.password?.message) {
       errorMessage = errors.password.message;
+    } else if (errors.unspecifiedError?.message) {
+      errorMessage = errors.unspecifiedError.message;
     }
 
     setErrorMessage(errorMessage);
-  }, [errors.email?.message, errors.password?.message, setErrorMessage]);
+  }, [errors.email?.message, errors.password?.message, errors.unspecifiedError?.message, setErrorMessage]);
+
+  useEffect(() => {
+    clearErrors("unspecifiedError");
+  }, [getValues("email"), getValues("password"), clearErrors]);
+
+  const onSubmit = async (value) => {
+    try {
+      const response = await axios.post("http://localhost:8000/api/auth/token/login/", {
```

Comment:
Could you move the api url to .env file and use it

Reply:
This endpoint returns just token. 
All the endpoints of API have to be moved to .env file?

Reply:
I mean instead of writing every time `http://localhost:8000/api/auth/token/login` we may create ` .env `file for the frontend app add base API url there and reuse it. ``${process.env.REACT_APP_BASE_API_URL}/auth/token/login/``

Reply:
Check it https://www.architect.io/blog/2022-08-16/react-environment-variables-developers-guide/

Reply:
Done